CONTRIBUTING.md

```diff
@@ -32,6 +32,18 @@ First of all, thank you for contributing to MeiliSearch! The goal of this docume
 $ composer install
 ```
 
+### Start MeiliSearch service <!-- omit in TOC -->
+
+In order to run all Unit and Feature tests, you need to start a MeiliSearch container or a local instance. See [Getting Started](https://github.com/meilisearch/MeiliSearch#getting-started)
+
+#### Docker (recommended) <!-- omit in TOC -->
+
+This will spin up a fresh MeiliSearch instance and will cleanly remove itself after using it.
+
+```bash
+$ docker run --rm -p 7700:7700 getmeili/meilisearch:latest ./meilisearch --no-analytics=true
+```
+
```

Comment:
This part is not needed in the CONTRIBUTING, since it's necessary for the users, so we put it in the REAMDE 🙂
https://github.com/meilisearch/meilisearch-laravel-scout#run-meilisearch-

Reply:
@curquiza should I rewrite the instruction in general, because this is written in a context of how to start the MeiliSearch instance to run tests?

Reply:
Not sure to understand what you mean.
In the CONTRIBUTING -> this helps the contributors (the Meili team or the external contributor) to use the package. How to run the test and the linter.
In the README -> This helps the user to use the library, so launching a MeiliSearch instance is a mandatory step. That's why we already put in the README (https://github.com/meilisearch/meilisearch-laravel-scout#run-meilisearch-). The contributors are already aware of this, they know how to run a MeiliSearch instance because they know how to use the library (at least the basic steps). That's why we don't repeat the step in the CONTRIBUTING file :)

Reply:
Understood :+1: I removed this part